src/reference/which-crates-work-with-wasm.md

```diff
@@ -0,0 +1,61 @@
+# Which Crates Will Work Off-the-Shelf with WebAssembly?
+
+It is easiest to list the things that do *not* currently work with WebAssembly;
+crates which avoid these things tend to be portable to WebAssembly and usually
+*Just Work*. A good rule of thumb is that if a crate supports embedded and
+`#![no_std]` usage, it probably also supports WebAssembly.
+
+## Things a Crate Might do that Won't Work with WebAssembly
+
+### C and System Library Dependencies
+
+There are no system libraries in wasm, so any crate that tries to bind to a
+system library won't work. Using C libraries will also probably fail to work,
+since wasm doesn't have a stable ABI for cross-language communication, and
+cross-language linking for wasm is very finicky.
```

Comment:
It might be good to add a note here that we would like to get this working, but it's an ongoing story. For example eventually with Clang that ships wasm support by default it "should simply be a matter of grabbing a sysroot". 

In any case, I think eventually we'll have support for this, the story's just not quite there yet